### PR TITLE
KOGITO-1446 maintain filter selections between page changes

### PR DESCRIPTION
**Pull request by AjayJagan**
- added changes to main filter state between page changes

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]
@AjayJagan I'm happy to merge this version, but as I mentioned to @Sara4994, I think we need a way to do this without having to pass state between screens and re-execute the query.

File: packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx [hunk 1]
```diff
@@ -51,60 +55,101 @@ const DataListComponent: React.FC<IOwnProps> = ({
...
+    ) {
+      data2.ProcessInstances.map((instance: any) => {
+        instance.isChecked = false;
+        instance.isOpen = false;
+      });
+    }
+    setInitData(data2);
+  }, [data2]);
+
+  if (loading1 || isLoading || loading2) {
```
Comment (nmirasch) [inline packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx]: @AjayJagan This var naming could be difficult to maintain. In order to improve maintainment , could be possible rename variables like data+ number, loading+number to a name that represents what's is exactly referring?

File: packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx [hunk 2]
```diff
@@ -51,60 +55,101 @@ const DataListComponent: React.FC<IOwnProps> = ({
...
-        <SpinnerComponent spinnerText="Loading process instances..." />
-      </Bullseye>
-    );
+  if (error1) {
+    setIsError(true);
+    return <ServerErrorsComponent message={error1} />;
   }
 
-  if (error) {
+  if (error2) {
```
Comment (nmirasch) [inline packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx]: @AjayJagan  same here with the error+number.

**Comment by nmirasch** [comment]
@AjayJagan It works fine now with remote data index service. Just added var renaming comments there.

File: packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx (hunk 2 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx]: sure, changed it

File: packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx (hunk 1 above)
Comment (AjayJagan) [reply packages/management-console/src/components/Organisms/DataListComponent/DataListComponent.tsx]: agreed. Both should have readable names.

**Review by nmirasch — APPROVED** [review]
Works fine and looks good, Thanks!

**Comment by cristianonicolai** [comment]
@AjayJagan please align with @Sara4994 regarding the approach used in https://github.com/kiegroup/kogito-apps/pull/173

**Comment by nmirasch** [comment]
@AjayJagan could you rebase?

**Comment by AjayJagan** [comment]
closing this PR and raising a new one with a different logic

**Pull request closed without merging**